api/admin/clients.js

```
@@ -0,0 +1,55 @@
+    if (req.method === 'OPTIONS') {
+        res.status(200).end();
+        return;
+    }
+
+    if (req.method !== 'GET') {
+        return res.status(405).json({ error: 'Method not allowed' });
+    }
+
+    if (!requireAdmin(req, res)) return;
+
+    const supabaseUrl = process.env.VITE_SUPABASE_URL;
```

ESLint

[error] 'process' is not defined.

(no-undef)